Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN rm -rf /go/src/github.com/openshift/csi-driver-shared-resource/examples
 RUN rm -f /go/src/github.com/openshift/csi-driver-shared-resource/vendor/k8s.io/apimachinery/pkg/util/managedfields/pod.yaml
 RUN make build
 
-FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4
```

Comment:
I think this needs to be updated to `ubi:9.4` - tests failing due to missing the `mount` command:

```
        E0506 11:02:26.024479       1 server.go:118] GRPC error: rpc error: code = Internal desc = failed to mount device: e2e-test-csi-driver-shared-resource-8ksnv-e2e-test-csi-driver-shared-resource-8ksnv-csi-312ee9d4ec3256014f4706131c4ce8a572cfa299188ddad473c67b7c61eeaa16 at /var/lib/kubelet/pods/5fe676ed-9792-4f2d-a969-617a9a28b8d3/volumes/kubernetes.io~csi/my-csi-volume/mount: mount failed: exec: "mount": executable file not found in $PATH
        Mounting command: mount
```